controllers/apps/components/internal/component_base_stateful.go

```diff
@@ -586,6 +586,11 @@ func (c *StatefulComponentBase) scaleOut(reqCtx intctrlutil.RequestCtx, cli clie
 		}
 	)
 
+	// sts's replicas=0 means it's starting not scaling, skip all the scaling work.
+	if *stsObj.Spec.Replicas == 0 {
```

Comment:
this bug is rather critical, is it exist in v0.5.x?

Reply:
It seems that it is existing in 0.5.x

Reply:
no, it's not existed in v0.5.x, that's a bug from full backup h-scale